meshnesh/android-level-up
# FT Create layout file for DetailActivity #155820645

# What does this PR do?

* Creates the layout for the DetailLayout activity.
* Allows DetailLayoutActivity to be launched from the MainActivity.

## How should this be manually tested?

* Clone the repo to your local machine.
* Open the project with Android Studio.
* Run an emulator or adb connection and run the app.
* Tap on one of the developers in the main page to see the details activity.

## What are the relevant pivotal tracker stories?

[Create layout file for DetailActivity](https://www.pivotaltracker.com/story/show/155820645)
[Launch the DetailActivity from the MainActivity](https://www.pivotaltracker.com/story/show/155820624)

### Screenshots/ GIF

![pr](https://user-images.githubusercontent.com/15923109/42140856-0a27f4ba-7dad-11e8-9479-cae0d0a00101.gif)
